### PR TITLE
lava-slave: update ser2net configuration file

### PR DESCRIPTION
**Pull request by aliceinwire**
on new versions of ser2net ser2net.conf configuration file as been deprecated and replaced by ser2net.yaml

Signed-off-by: Alice Ferrazzi <alice.ferrazzi@miraclelinux.com>

### CODE REVIEW

**Comment by aliceinwire** [comment]
adding here under the resulting output of ser2net.yaml:
```
%YAML 1.1
---
connection: &con63001
  accepter: telnet(rfc2217),tcp,63001
  enable: on
  connector: serialdev,/dev/meson-gxl-s905x-libretech-cc-01,115200n81,local
  options:
    max-connections: 10
connection: &con63002
  accepter: telnet(rfc2217),tcp,63002
  enable: on
  connector: serialdev,/dev/meson-gxbb-nanopi-k2-01,115200n81,local,nobreak,option_2
  options:
    max-connections: 10
connection: &con63003
  accepter: telnet(rfc2217),tcp,63003
  enable: on
  connector: serialdev,/dev/meson-gxbb-nanopi-k3-01,115200n81,local,nobreak,option_1
  options:
    max-connections: 10
```

**Comment by aliceinwire** [comment]
Fixed the missing: 
```                
ser2net_ports[worker_name] += 1
```

File: lavalab-gen.py [hunk 1]
```diff
@@ -731,32 +731,30 @@ def main():
...
+                    ser2net_yaml_line= "  connector: keepopen(retry-time=2000,discard-badwrites),serialdev,/dev/%s,%dn81,local" % (board_name, baud)
+                    if "ser2net_options" in uart:
+                        for ser2net_yaml_option in uart["ser2net_options"]:
+                            ser2net_yaml_line += " %s" % ser2net_yaml_option
                 else:
-                    fp.write("  connector: serialdev,/dev/%s,%dn81,local\n" % (board_name, baud))
+                    ser2net_yaml_line = "  connector: serialdev,/dev/%s,%dn81,local" % (board_name, baud)
+                    if "ser2net_options" in uart:
+                        for ser2net_yaml_option in uart["ser2net_options"]:
+                            ser2net_yaml_line += ",%s" % ser2net_yaml_option
```
Comment (montjoie) [inline lavalab-gen.py]: lines 745-747 and 750-752 are the same, they could be put without an indent less just before line 753

So
if keepopen
    line=
else
   line =
if ser2netoptions

please

Comment (montjoie) [reply lavalab-gen.py]: I just saw a difference, line 752 has an extra "," vs line 747. I think both needed a ","

Comment (aliceinwire) [reply lavalab-gen.py]: updated

**Pull request merged**